### ISSUE TITLE
Bard AI images

### ISSUE THREAD

**Issue by Aaarkay1** [josStorer/chatGPTBox#414]
**Is your feature request related to a problem? Please describe.**
No images for Bard AI is shown. I understand BARD was added recently, but still I thought of reminding the developer. 

**Additional context**
![1](https://github.com/josStorer/chatGPTBox/assets/63100754/9453806b-8c5f-4750-b292-e9158c4e55ea)

**Comment by Wladefant** [comment]
Yes I agree, I think an implementation similar to this extension is good: 
![image](https://github.com/josStorer/chatGPTBox/assets/79611093/44eeaeae-a178-41a0-b9cf-032e11dac7fb)
